### ISSUE TITLE
Automate GlobaLeaks SEO friendlyness tests with Google Lighthouse

### ISSUE THREAD

**Issue by fpietrosanti** [globaleaks/globaleaks-whistleblowing-software#2700]
**Current behavior**
Google provide https://developers.google.com/web/tools/lighthouse/ that does audits for SEO, performance, accessibility, progressive web apps, and more.

**Expected behavior**
An Audit and fixing should be done manually using lighthouse to evaluate non compliance and fix it, then integrate it's auditing in continuous integration testing as per #2699

**Comment by fpietrosanti** [comment]
Other than lighthouse chrome extension, useful also to check with google search console for suggestions https://search.google.com/search-console/welcome

**Comment by AnXh3L0** [comment]
The Lighthouse testing has been added to the features/gh-actions-fix branch, and it's now working successfully, as per the last successful action https://github.com/globaleaks/globaleaks-whistleblowing-software/actions/runs/15729245161/job/44326242560#step:10:1929. Further improvements are being made to display the results in the main README file.

**Comment by evilaliv3** [comment]
@AnXh3L0: i just found https://roquec.com/articles/2023-10-24-lighthouse-automated-reporting/

When you prefer, do you mind giving it a try?

I think that, when running on the 'stable' branch, we could use this recipe to publish on a gist the result of lighthouse.

If it work, then we could edit the README.MD to show the badges as in https://github.com/roquec/portfolio/blob/main/README.md

Kudos to @roquec for this very nice work!

**Comment by AnXh3L0** [comment]
@evilaliv3 I have pushed a commit in the following branch, with an implementation identical to the one provided in your comment https://github.com/globaleaks/globaleaks-whistleblowing-software/tree/feature/lighthouse-audit-gist. The gist update and publishing will only work on the stable branch, but we can modify it for now to any branch to see the actual process happening.

**Comment by AnXh3L0** [comment]
Based on the latest changes made to the branch above, we will now push new reports directly to a new repository instead of using a gist, and will point the badges and the links directly to the raw content of the generated report.html file. This will ensure that the scope is refined and provides a better security model.

**Comment by evilaliv3** [comment]
I confirm that all the goals pursued within this ticket are satisfied and we could close the ticket.

I already proceeded integrating this work on the stable branch and added same badges to the documentation of the [Quality Assurance](https://docs.globaleaks.org/en/stable/technical/quality-assurance/index.html)

Below for archival purposes some visual representation of the current status.

<img width="471" height="686" alt="Image" src="https://github.com/user-attachments/assets/9e9a754a-39e3-4528-98b9-c9bb110fb946" />

<img width="1920" height="1168" alt="Image" src="https://github.com/user-attachments/assets/78fe6a60-2125-4f55-bf4c-05ee9f386207" />
